Sourcery/Generating/Template/JavaScript/ejsbundle.js

```diff
@@ -1,7 +1,25 @@
 (function e(t,n,r){function s(o,u){if(!n[o]){if(!t[o]){var a=typeof require=="function"&&require;if(!u&&a)return a(o,!0);if(i)return i(o,!0);var f=new Error("Cannot find module '"+o+"'");throw f.code="MODULE_NOT_FOUND",f}var l=n[o]={exports:{}};t[o][0].call(l.exports,function(e){var n=t[o][1][e];return s(n?n:e)},l,l.exports,e,t,n,r)}return n[o].exports}var i=typeof require=="function"&&require;for(var o=0;o<r.length;o++)s(r[o]);return s})({1:[function(require,module,exports){
 (function (global){
- var ejs = require('ejs');
- global.content = ejs.render(template, templateContext);
+  
+  let ejs = require('ejs');
+  
+  function renderTemplate(source, context, filename) {
+    let fn = ejs.compile(source, {client: true, filename});
+    
+    return fn(context, null, function(path, d) {
+      let template = include(path);
+      return renderTemplate(template, d, path);
+     }, function(error,source,filename,lineno) {
+      rethrow({message: error.message}, source, filename, lineno);
+    });
+  }
+  
+  try {
+    global.content = renderTemplate(template, templateContext, templateName);
+  } catch (e) {
+    global.content = "// Error compiling/rendering EJS template\n// " + e.message;
```

Comment:
maybe better to rethrow instead of setting content? sourcer, fileame and lineno can be optional.